rudder-web/src/main/scala/com/normation/rudder/appconfig/ConfigService.scala

```diff
@@ -110,19 +110,19 @@ trait ReadConfigService {
    */
   def rudder_store_all_centralized_logs_in_file(): Box[Boolean]
 
-
   /**
    * Compliance mode:
    * - "compliance": full compliance mode, where "success" execution reports are sent
    *   back from node to server, and taken into account for compliance reports,
    * - "error_only": only error and repaired are going to the server.
    */
-  def rudder_compliance_mode(): Box[(String,Int)] = {
+  def rudder_compliance_mode(): Box[ComplianceMode] = {
     for {
         name <- rudder_compliance_mode_name
+        modeName <- ComplianceModeName(name)
         period <- rudder_compliance_heartbeatPeriod
     } yield {
-      (name,period)
+      ComplianceMode(modeName,period, true)
```

Comment:
In fact it has no meaning to have an override for global compliance mode, so we could set any value ...

I have to made a design choice here: 

either only have one 'ComplianceMode' class with 3 param ( name, hearbeat, override)
or 1 trait 'complianceMode' with 2 param (name, heartbeat) and 2 sublcass GlobalComplianceMode(nothing more than complianceMode), NodeComplianceMode with a third parameter (overrides) (In fact you can see an example of this in commented section on which you said "Why the comments ??" ) 


I don't know which one would be better ???